lib/appmap/service/config_analyzer.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module AppMap
+  module Service
+    class ConfigAnalyzer
+      attr_reader :config_error
+
+      def initialize(config_file)
+        @config_file = config_file
+        @config = load_config
+      end
+
+      def app_name
+        @config.to_h[:name] if is_present?
+      end
+
+      def is_present?
```

Comment:
You can just use `present?` instead of `is_present?`, `is` implies `?`.

Reply:
Right! 👍